File: app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/title/TitleActivityHandlers.java
package hm.orz.chaos114.android.tumekyouen.modules.title;

import android.view.View;

/**
 * TitleActivityのHandlerクラス。
 */
public interface TitleActivityHandlers {
    void onClickStartButton(View view);

    void onClickGetStage(View v);

    void onClickCreateStage(View v);

    void onClickConnectButton(View view);

    void onClickSyncButton(View view);

    void switchPlayable(View view);

    void onClickPrivacyPolicy(View view);
}

File: app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/title/TitleActivityHandlers.kt
package hm.orz.chaos114.android.tumekyouen.modules.title

import android.view.View

interface TitleActivityHandlers {
    fun onClickStartButton(view: View)

    fun onClickGetStage(v: View)

    fun onClickCreateStage(v: View)

    fun onClickConnectButton(view: View)

    fun onClickSyncButton(view: View)

    fun switchPlayable(view: View)

    fun onClickPrivacyPolicy(view: View)
}
